.github/workflows/Dockerfile.dev

```diff
@@ -31,4 +31,12 @@ RUN wget -c https://musl.cc/x86_64-linux-musl-cross.tgz && \
     rm ./x86_64-linux-musl-cross.tgz && \
     rm ./aarch64-linux-musl-cross.tgz
 
+### armhf deps
```

Comment:
I'll need to rebuild/republish the build image, right?

Reply:
right, im thinking keep the deps in one development image. 👍🏼